Cargo.toml

```diff
@@ -15,7 +15,7 @@ atty = "0.2.14"
 base64 = "0.13.0"
 billboard = "0.1.0"
 binary-install = "0.0.3-alpha.1"
-chrome-devtools-rs = { version = "0.0.0-alpha.1", features = ["color"] }
+chrome-devtools-rs = { git = "https://github.com/cloudflare/chrome-devtools-rs", branch="fix/devlinenums" }
```

Comment:
Can we wait to merge this until https://github.com/cloudflare/chrome-devtools-rs/pull/48 is merged?

Reply:
Sure- btw, that's an abandoned-ish repo which used to be admin'd by Avery Harnish. I'm the admin now so I can just shove my changes through, it would be nice to get another set of eyes on it though.